mix.exs

```diff
@@ -53,6 +53,7 @@ defmodule Ret.Mixfile do
       {:plug_cowboy, "~> 2.0"},
       {:distillery, "~> 2.0"},
       {:peerage, "~> 1.0"},
+      {:hackney, "~> 1.16.0", override: true},
```

Comment:
Do we actually need to specify this override in mix.exs? I had this working on my system with just the lockfile changes (resulting from just installing the latest version of the package, not hand editing it or anything)... Not super familiar with standard practice for elixir stuff but "override" feels like a scary thing.

Reply:
IMO, it's better to be explicit in the mix.exs file, than to be implicit in the mix.lock file, because we often don't manually read diffs of the mix.lock file.